src/plugins/controls/server/options_list/options_list_cluster_settings_route.ts

```diff
@@ -20,6 +20,12 @@ export const setupOptionsListClusterSettingsRoute = ({ http }: CoreSetup) => {
     .addVersion(
       {
         version: '1',
+        security: {
+          authz: {
+            enabled: false,
+            reason: 'This route is opted out from authorization',
```

Comment:
```suggestion
            reason: 'This endpoint is opted out from authorization because it does not take a query, params, or a body, so there is no chance of leaking info.',
```

almost verbatim from the inline comments below. 😆